Monika After Story/game/script-ch30.rpy

```diff
@@ -1778,14 +1778,21 @@ label ch30_reset:
         #TODO: Fix the game unlock/is unlocked system to account for conditions like these
         #mas_isGameUnlocked should NOT return True if we're failing this condition because we use it elsewhere
```

Comment:
Unreleated to the rest of the PR, but since we're here, can we remove this `TODO`? This was actually done when I changed the game system a while ago